lms/envs/common.py

```diff
@@ -379,6 +379,9 @@
     # Certificates Web/HTML Views
     'CERTIFICATES_HTML_VIEW': False,
 
+    # Batch-Generated Certificates from Instructor Dashboard
+    'CERTIFICATES_INSTRUCTOR_GENERATION': False,
```

Comment:
One question, would someone want to have CERTIFICATES_HTML_VIEW turned on/off separately than CERTIFICATES_INSTRUCTOR_GENERATION?